app/uk/gov/hmrc/economiccrimelevyregistration/models/Registration.scala

```diff
@@ -47,6 +48,13 @@ final case class Registration(
 ) {
   def otherEntityJourneyData: OtherEntityJourneyData =
     optOtherEntityJourneyData.getOrElse(OtherEntityJourneyData.empty())
+
+  def isRegistration: Boolean =
+    (entityType, registrationType) match {
+      case (Some(value), _) if EntityType.isOther(value) => true
+      case (_, Some(Amendment))                          => true
```

Comment:
Should an amendment be classed as isRegistration?

Reply:
hm...it's used to differentiate between a registration and subscription